src/components/molecules/NewsletterForm/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { FormProvider } from 'react-hook-form'
```

Comment:
Another tangled component, please divide it into smaller chunks so it could be readable fasade of compoentns like

`isSuccess && <NewsletterMemberInformation />`
`!isSuccess && <JoinNewsletter />`